src/cipher/encoder/encoder.go

```diff
@@ -765,6 +770,10 @@ func (d *decoder) value(v reflect.Value) error {
 		nFields := v.NumField()
 		for i := 0; i < nFields; i++ {
 			ff := t.Field(i)
+			// Skip unexported fields
```

Comment:
This doesn't impact any existing message structs does it?

Reply:
No that I know . Thing is that unexported fields may not be read . Now they're skipped using `enc:-`.